crawler/Cargo.toml

```diff
@@ -0,0 +1,60 @@
+[package]
+name = "snarkos-crawler"
+version = "2.0.0"
+authors = [ "The Aleo Team <hello@aleo.org>" ]
+description = "A decentralized operating system"
+homepage = "https://aleo.org"
+repository = "https://github.com/AleoHQ/snarkOS"
+keywords = [
+    "aleo",
+    "cryptography",
+    "blockchain",
+    "decentralized",
+    "zero-knowledge"
+]
+categories = [ "cryptography", "operating-systems" ]
+license = "GPL-3.0"
+edition = "2018"
+
+[dependencies.snarkos]
+path = ".."
+features = ["test"]
```

Comment:
why are we using this feature here? presumably only so that there is no storage path clash with the proper node?

Reply:
This is actually left over from the synthetic node refactor, it runs fine without 👍

Reply:
cool; as long as there is no persistent storage associated with the crawler, there will be no clash